### PR TITLE
Add support symfony 5, add support php74, drop support php71

### PR DESCRIPTION
**Pull request by veewee**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | yes
| Deprecations? | no
| Documented?   | no
| Fixed tickets |

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 7.1
   - 7.2
   - 7.3
+  # - 7.4 // Not available yet on Travis
```
Comment (drupol) [inline .travis.yml]: I think you can use `7.4snapshot`

**Comment by drupol** [comment]
Nice, eagerly waiting for this.

File: .travis.yml (hunk 1 above)
Comment (veewee) [reply .travis.yml]: Thanks! Will try

File: src/Console/Command/ConfigureCommand.php [hunk 2]
```diff
@@ -64,7 +64,7 @@ protected function configure(): void
     /**
      * @return int|void
      */
-    public function execute(InputInterface $input, OutputInterface $output)
+    public function execute(InputInterface $input, OutputInterface $output): int
```
Comment (Landerstraeten) [inline src/Console/Command/ConfigureCommand.php]: Nice!

**Pull request merged**